app/src/main/java/com/example/sherryuan/greenthumb/persistence/PlantDao.java

```diff
@@ -0,0 +1,42 @@
+package com.example.sherryuan.greenthumb.persistence;
+
+import android.arch.persistence.room.Dao;
+import android.arch.persistence.room.Delete;
+import android.arch.persistence.room.Insert;
+import android.arch.persistence.room.OnConflictStrategy;
+import android.arch.persistence.room.Query;
+
+import java.util.List;
+
+import io.reactivex.Flowable;
+
+/**
+ * Created by sherryuan on 2018-08-29.
+ */
+
+@Dao
```

Comment:
What's a `Dao`?

Reply:
it stands for Data Access Object, classes annotated with `@Dao` have methods that act as SQL queries